fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/LabelExistsVersionExceptionMapper.java

```diff
@@ -41,8 +41,9 @@
 
     @Override
     public Response toResponse(final LabelExistsVersionException e) {
-        LOGGER.debug("LabelExistsVersionException intercepted by LabelExistsVersionExceptionMapper"
-                    + (e.getMessage() != null ? ": " + e.getMessage() : "."), e);
+        LOGGER.error("LabelExistsVersionException intercepted by LabelExistsVersionExceptionMapper"
```

Comment:
Instead of String concatenation in this message with `+`, please use the template syntax of `{}`.